salt/states/virt.py

```diff
@@ -668,8 +668,9 @@ def defined(
                 stop_on_reboot=stop_on_reboot,
                 host_devices=host_devices,
             )
-            ret["changes"][name] = status
+            ret["changes"] = status
```

Comment:
Removing `name` from the `changes` dictionary would be an unexpected change for users who are parsing the return data. This does not seem related to the bug fix. Is this  a necessary change?

Reply:
well, I can put it back it just is easier to handle this way. I'll fix it on monday then